controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -118,5 +118,48 @@ func configureServiceMeshFeatures(s *feature.FeaturesInitializer) error {
 		s.Features = append(s.Features, metricsCollection)
 	}
 
+	var err error
+	var cfMaps *feature.Feature
+	if cfMaps, err = feature.CreateFeature("shared-config-maps").
+		For(s.DSCInitializationSpec).
+		WithResources(servicemesh.ConfigMaps).
+		Load(); err != nil {
+		return err
+	}
+	s.Features = append(s.Features, cfMaps)
+
+	var extAuthz *feature.Feature
+	if extAuthz, err = feature.CreateFeature("service-mesh-control-plane-setup-external-authorization").
+		For(s.DSCInitializationSpec).
+		Manifests(
+			path.Join(rootDir, feature.AuthDir, "auth-smm.tmpl"),
+			path.Join(rootDir, feature.AuthDir, "base"),
+			// path.Join(rootDir, feature.AuthDir, "rbac"),
```

Comment:
@bartoszmajsak Looks like we don't need this for the time being?

Reply:
After looking at it I am not quite sure where this came from :) I assume SAR calls are working fine on your end, and generally speaking those roles should be granted by the operator itself, so I would say let's ditch it.

Reply:
Removed.